app/views/user_case_infos/edit.html.erb

```diff
@@ -5,3 +5,24 @@
 
 <%= link_to 'Show', @user_case_info %> |
 <%= link_to 'Back', user_case_infos_path %>
+
```

Comment:
Since the `edit.html.erb` page re-renders `_form.html.erb`, these changes aren't needed (you end up with the same fields listed twice on the page)